### PR TITLE
[DOCS] Increments version from 0.11.0 to 0.11.1

### PR DESCRIPTION
**Pull request by patkenneally**
* **Tickets addressed:** NA
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description
Increments version from 0.11.0 to 0.11.1

## Verification
CI process tests run

## Documentation
No new documentation to create and no updates needed.

## Future work
NA

### CODE REVIEW

**Comment by camargo** [comment]
I have been bad about updating these. Can you also update the version of the default UI views pease: 
https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/postgres-init-db/data/ui/views/example_0.json#L7
https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/postgres-init-db/data/ui/views/example_1.json#L7

File: command-expansion-server/test/utils/MissionModel.ts [hunk 1]
```diff
@@ -6,11 +6,11 @@ export async function uploadMissionModel(graphqlClient: GraphQLClient): Promise<
   /*
    * Insert a mission model
    */
-  const bananaNationMissionModelUrl = new URL('../inputs/banananation-0.11.0-SNAPSHOT.jar', import.meta.url);
+  const bananaNationMissionModelUrl = new URL('../inputs/banananation-0.11.1-SNAPSHOT.jar', import.meta.url);
```
Comment (mattdailis) [inline command-expansion-server/test/utils/MissionModel.ts:9]: I think this needs to match the jar filename in the inputs directory.

Comment (mattdailis) [reply command-expansion-server/test/utils/MissionModel.ts:9]: https://github.com/NASA-AMMOS/aerie/tree/b03c8c87c95eee56b2baaca1c313a4fc14bd134f/command-expansion-server/test/inputs

Comment (patkenneally) [reply command-expansion-server/test/utils/MissionModel.ts:9]: Thanks for the heads up on this. I've also created a ticket to remove this and grab the jar from the bananation build directory.

**Review by camargo — APPROVED** [review]

**Review by mattdailis — APPROVED** [review]

**Comment by patkenneally** [comment]
> 

Thanks for the heads up on these ones. I missed them with my find on "0.11.0"

**Comment by patkenneally** [comment]
> > 
> I have been bad about updating these. Can you also update the version of the default UI views pease:
> https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/postgres-init-db/data/ui/views/example_0.json#L7
> https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/postgres-init-db/data/ui/views/example_1.json#L7

Thanks for the heads up on these ones. I missed them with my find on "0.11.0"

**Pull request merged**